fiona/_drivers.pyx

```diff
@@ -147,7 +147,7 @@ cdef class GDALEnv(object):
                 val_b = ('ON' if val else 'OFF').encode('utf-8')
             val_c = val_b
             CPLSetThreadLocalConfigOption(key_c, val_c)
-            log.debug("Option %s=%s", key, CPLGetConfigOption(key_c, NULL))
+            val_d = CPLGetConfigOption(key_c, NULL)
```

Comment:
@youngpm we have one test for these debug messages. Can you restore the call to `log.debug()`, please? If you do, your PR will pass :)